PIL/TiffTags.py

```diff
@@ -417,23 +424,25 @@ def _populate():
 # 393:	case TIFFTAG_INKNAMES:
 
 # some of these are not in our TAGS_V2 dict and were included from tiff.h
+# Anything included here needs to have the correct type in TAGS_V2 above
 
 LIBTIFF_CORE = {255, 256, 257, 258, 259, 262, 263, 266, 274, 277,
                 278, 280, 281, 340, 341, 282, 283, 284, 286, 287,
                 296, 297, 321, 320, 338, 32995, 322, 323, 32998,
                 32996, 339, 32997, 330, 531, 530, 301, 532, 333,
                 # as above
-                269  # this has been in our tests forever, and works
+                269,  # this has been in our tests forever, and works
+                318, # Whitepoint, Specific test for it
+                319, # Primary Chromaticities
                 }
 
-LIBTIFF_CORE.remove(320)  # Array of short, crashes
-LIBTIFF_CORE.remove(301)  # Array of short, crashes
-LIBTIFF_CORE.remove(532)  # Array of long, crashes
+LIBTIFF_CORE.remove(330)  # subifd, requires extra support for uint64 payload
```

Comment:
Given that https://github.com/python-pillow/Pillow/pull/5120 blocked the tag when saving with libtiff, I don't think this is needed anymore.